### PR TITLE
mgmt sample update 

### PR DESCRIPTION
**Pull request by xseeseesee**
Fix: [Azure/azure-libraries-for-java#1062](https://github.com/Azure/azure-libraries-for-java/issues/1062)

### CODE REVIEW

File: sdk/management/samples/src/main/java/com/azure/management/compute/samples/ConvertVirtualMachineToManagedDisks.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+     *
+     * @param azure instance of the azure client
+     * @return true if sample runs successfully
+     */
+    public static boolean runSample(Azure azure) {
+        final String linuxVMName = azure.sdkContext().randomResourceName("VM1", 15);
+        final String rgName = azure.sdkContext().randomResourceName("rgCOMV", 15);
+        final String userName = "tirekicker";
+        // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Serves as an example, not for deployment. Please change when using this in your code.")]
+        final String password = "12NewPA$$w0rd!";
```
Comment (weidongxu-microsoft) [inline sdk/management/samples/src/main/java/com/azure/management/compute/samples/ConvertVirtualMachineToManagedDisks.java:37]: Please randomize at least part of it (I think Tanyi used the `randomeResourceName` for this purpose. May worth put to Util class if needed in more places).

**Comment by weidongxu-microsoft** [comment]
Forget to mention, try this https://github.com/Azure/azure-sdk-for-java/pull/10488 before commit json recording. For now it should erase storage keys.

**Comment by xccc-msft** [comment]
> Forget to mention, try this #10488 before commit json recording. For now it should erase storage keys.

I tried this by setting the path to `samples\src\test\resources\session-records\`. It seems not storage key in compute/graph records.

File: sdk/management/samples/src/main/java/com/azure/management/compute/samples/ConvertVirtualMachineToManagedDisks.java (hunk 1 above)
Comment (ChenTanyi) [reply sdk/management/samples/src/main/java/com/azure/management/compute/samples/ConvertVirtualMachineToManagedDisks.java:37]: Here https://github.com/Azure/azure-libraries-for-java/pull/863/files. Could add to utils if needed.

**Comment by weidongxu-microsoft** [comment]
Forget to mention... Please run spotbugs on samples. Should only report on "throw Exception".

**Pull request merged**